AFTests/AssetsTests/PartialWatchLists.cs

```diff
@@ -181,7 +181,8 @@ public partial class AssetsTest
 
                 foreach (string assetId in entity.AssetIDsList)
                 {
-                    parsedMatch.AssetIds.Should().Contain(assetId);
```

Comment:
We should investigate why there would be an empty assetId instead of having a workaround so that the tests would pass.

Reply:
when AssetIDs is empty, the service returns NULL, In azure the assetIds are stored as a comma-seperated values string, and splitting an empty string results in a list of 1 empty string value. Will fix in WatchListEntity getters and setters in the new branch.